lib/wrappers/genesisProtocol.ts

```diff
@@ -961,138 +985,70 @@ export interface RedeemEventResult {
 
 export interface GenesisProtocolParams {
   /**
-   * the absolute vote percentages bar
-   * Must be greater than zero.
+   * The percentage of the absolute vote that must be exceeded to result in a win.
+   * Must be between 0 and 100.
    * Default is 50.
```

Comment:
Again , define the defaults values  externally  and point from the comment to the defaults definition..
By doing that you , there will be no need to update all comments when updating the defaults.

Reply:
The comments are already only in one place.  I don't worry about the comments in the migration script.

But still, your suggestion is one that I have also considered several times.  It depends on the defaults for DAO.new being necessarily the same as the defaults for the Genesis DAO.  For now at least, that is the case, but may not always be.

In any case, there are issues with including code in the deployment script, and babel: (https://github.com/daostack/arc.js/issues/156).  I would have to figure out how to avoid those issues, and it is a pain to do as it requires testing migration in the context of an application.

There could also result problems with auto-generated API documentation, depending on how I solve the problem with babel.

So I am not inclined right now to make this change.

Reply:
is it no as simple as creating a file (naming something like genesisDefaultParams.js and import that fro the migration file ?

Reply:
No, it isn't that simple, it has the implications I described above.  However, I think it is worth figuring out how to do this.  There is another bit that has also become redundant because of this problem, and I don't like it either.  So I've created a ticket on it:  https://github.com/daostack/arc.js/issues/171

Reply:
One option that would be easy would be to put the data into a .json file rather than javascript.